tsuru/client/plugin_test.go

```diff
@@ -24,6 +25,57 @@ func (s *S) TestPluginInstallInfo(c *check.C) {
 	c.Assert(PluginInstall{}.Info(), check.NotNil)
 }
 
+func (s *S) TestPluginInstallWithManifest(c *check.C) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "fakeplugin")
+	}))
+	defer ts.Close()
+	ts2 := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		urls := make(map[string]string)
+		urls["darwin/amd64"] = ts.URL
```

Comment:
as this is a test, I think it is enough to add a single platform on this map, based on GOOS/GOARCH.

What do you think?
Also, what do you think about responding with a plain json (instead of marshaling a struct)?

something direct and short, like:
```go
ts2 := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
	w.Header().Set("Content-Type", "application/json")
	urls := make(map[string]string)
	jsonResp = fmt.Sprintf(`{
  "SchemaVersion":"1.0",
  "Metadata": {"Name": "myplugin", "Version": "0.33.1"},
  "URLPerPlatform": {
    "%s/%s": "%s"
  }
}`, runtime.GOOS, runtime.GOARCH, ts.URL)
```

Reply:
Much better! Makes perfect sense 👍🏻